dist/datasource.js

```diff
@@ -13,6 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
```

Comment:
You write this change https://github.com/dkhwangbo/druidplugin/pull/6/commits/aedd96e18330867cfafd72aaefdb84799b3ebf28 but not applied. I think you need rebuild

Reply:
I forgot😲. After building I committed again. 924abf1